### PR TITLE
Upgrade page

### PR DESCRIPTION
**Pull request by kozaktar**
Added upgrade page to present selection of service plans to the user:
-A card can be set to selected with "selected" prop inactivating the button and adding a solid border.
-Buttons have on hover effect changing button background to primary color (orang) and changing text color to white  

image

### CODE REVIEW

**Comment by WhoaNellie** [comment]
Not sure how the eslintcache and the debug log keep sneaking back into our PRs but you might want to delete those files before merging

Otherwise looks good!

**Review by WhoaNellie — APPROVED** [review]

File: client/src/components/upgradeCard/UpgradeCard.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          classes={{ root: classes.buttonRoot }}
+          disabled={selected}
+        >
+          {selected?"Active":"Upgrade"}
+        </Button>
+      </div>
+      <Divider />
+      <div className={`${classes.root} ${classes.listContainer}`}>
+        <List>
+          {featuresList.map(item=>(<PlanListItem itemText={item}/>))}
```
Comment (alvyjudy) [inline client/src/components/upgradeCard/UpgradeCard.js:93]: make sure you pass a key attribute to the <PlanListItem/> here

File: client/src/components/upgradeCard/UpgradeCard.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    borderStyle:"solid",
+    borderWidth:"2px",
+    color:theme.palette.primary.main
+  }
+}));
+
+const UpgradeCard = ({ title, titleColor, featuresList, price, selected }) => {
+  const classes = useStyles();
+  const elevation=selected?20:8;
+  console.log(featuresList)
```
Comment (alvyjudy) [inline client/src/components/upgradeCard/UpgradeCard.js:72]: Logging not needed upon merging

**Review by alvyjudy — CHANGES_REQUESTED** [review]
The code looks good to me! Great work the UI is amazing! See comments for small suggestion.

Make sure to sync this branch with the main dev branch as a lot has changed. After doing that the .eslintcache and debug log should disappear and you should be good to merge.

File: client/src/pages/UpgradePage.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    color: "#909090"
+  },
+  cardContainer:{
+    marginTop:theme.spacing(4)
+  }
+}));
+
+const UpgradePage = () => {
+  const classes = useStyles();
+  const plan= "free basic";
```
Comment (dproc96) [inline client/src/pages/UpgradePage.js:37]: I'm assuming the plan is to get this value from the back-end when it's integrated in a future PR?

**Review by dproc96 — APPROVED** [review]
I would echo Di's comments, but this is good to go once those changes are made

**Pull request merged**